.env.dev

```diff
@@ -5,3 +5,4 @@ API_BASE_URL=https://eqip-prototype-api-dev.fr.cloud.gov
 HASH_ROUTING=
 ALLOW_2FA_RESET=1
 DISABLE_2FA=1
+FLUSH_STORAGE=1
```

Comment:
When someone logs in to the system this should flush any of the previous data associated with the account.